server/firebase.js

```diff
@@ -1,3 +1,9 @@
+/**
+ * Common firebase functions ???
```

Comment:
Yup, any firebase interactions are handled in this file and common functions are exported for use elsewhere.